src/main/java/org/littil/api/contact/service/Contact.java

```diff
@@ -0,0 +1,19 @@
+package org.littil.api.contact.service;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+public class Contact {
+    private UUID id;
+
+    private UUID recipient;
+
+    private LocalDateTime contactDate;
+
+    private String medium;
```

Comment:
Andere naam voor medium ?

Reply:
we hadden dezelfde discussie al; dit is de naam die daar uit kwam; als je een beter label hebt dan plak ik dat erop anders is ie resolved.

Reply:
Ga niet opnieuw discussie starten. weet ook niet zo snel een beter label.